src/contracts/proxies/HaiProxy.sol

```diff
@@ -23,13 +28,7 @@ contract HaiProxy is HaiOwnable2Step, IHaiProxy {
   /// @inheritdoc IHaiProxy
   function execute(address _target, bytes memory _data) external payable onlyOwner returns (bytes memory _response) {
     if (_target == address(0)) revert TargetAddressRequired();
```

Comment:
This check isn't really required as OpenZeppelin's `Address` library would revert as well with an `address(0)` argument (since no code can be found there).